packages/tooltip/src/Tooltip.tsx

```diff
@@ -159,6 +170,7 @@ function Tooltip({
   trigger,
   variant = Variant.Light,
   triggerEvent = TriggerEvent.Hover,
+  enabled = true,
```

Comment:
What happens if a tooltip is shown and then enabled is set to false? It should hide the tooltip at that point right?

We need a test for this too?

Reply:
The way it works now is that it will close and remain closed on the next hover or click event (based on the `triggerEvent`).

Reply:
I feel like if a user sets enabled to false they expect the tooltip to not exist at that point in time. I.e. the JSX that is being rendered in the tooltip content may now be in a state that could be broken?

Reply:
I suppose that point is fair. I will say that the JSX is always being based on the same state as it was prior, and shouldn't be all that brittle. I'll make the change though, since I think the case you're describing is fair.

Reply:
Alright – changed the behavior here, and added a test for that behavior!